db/migrate/20170529142557_unify_container_definition_and_container.rb

```diff
@@ -0,0 +1,104 @@
+class UnifyContainerDefinitionAndContainer < ActiveRecord::Migration[5.0]
+  class ContainerDefinition < ActiveRecord::Base
+    has_one :container
+    self.inheritance_column = :_type_disabled
+  end
+
+  class Container < ActiveRecord::Base
+    belongs_to :container_definition
+    self.inheritance_column = :_type_disabled
+  end
+
+  def up
+    add_column :containers, :image,              :string
+    add_column :containers, :image_pull_policy,  :string
+    add_column :containers, :memory,             :string
+    add_column :containers, :cpu_cores,          :float
+    add_column :containers, :container_group_id, :bigint
+    add_column :containers, :privileged,         :boolean
+    add_column :containers, :run_as_user,        :bigint
+    add_column :containers, :run_as_non_root,    :boolean
+    add_column :containers, :capabilities_add,   :string
+    add_column :containers, :capabilities_drop,  :string
+    add_column :containers, :command,            :text
+
+    say_with_time("Copying over columns from container_definition to container") do
+      Container.all.each do |container|
+        container_definition = ContainerDefinition.find(container.container_definition_id)
+        container.update!(
+            :image              => container_definition.image,
+            :image_pull_policy  => container_definition.image_pull_policy,
+            :memory             => container_definition.memory,
+            :cpu_cores          => container_definition.cpu_cores,
+            :container_group_id => container_definition.container_group_id,
+            :privileged         => container_definition.privileged,
+            :run_as_user        => container_definition.run_as_user,
+            :run_as_non_root    => container_definition.run_as_non_root,
+            :capabilities_add   => container_definition.capabilities_add,
+            :capabilities_drop  => container_definition.capabilities_drop,
+            :command            => container_definition.command
+        )
+      end
+    end
+
+    remove_column :containers, :container_definition_id
+    drop_table :container_definitions # need to take care of :dependent => destroy relats? (like ccustom attributes)
```

Comment:
I dont think they have `custom_attributes` but they have `container_env_vars` and `container_port_configs` that needs to be ported too somehow.